docs/cel2/notices/l2-migration.md

```diff
@@ -1,21 +1,95 @@
-# Preparing for the L2 migration
-
-This page outlines breaking changes related to the Celo L2 migration. This migration requires changes by node operators on the different Celo networks.
+# Preparing for the L2 hardfork
 
 :::info
-This page will be updated continuously with information on upgrade runbooks and timelines as they come.
+This page will be kept updated with key information about the hardfork.
 
 * The Alfajores testnet has been migrated on block **26384000**, September 26, 2024.
 * The Baklava testnet has been migrated on block **28308600**, February 20, 2025.
 * Mainnet will be migrated on block **31056500**, March 25, 2025.
 
 :::
 
-## For node operators
+Node operators should review this page along with [Migrating a Celo L1 Node](../operators/migrate-node.md) and [Running a Celo Node](../operators/run-node.md).
+
+Besides that, the two main steps to prepare for the hardfork are
+
+1. Upgrading L1 nodes
+2. Running a pre-migration (or choosing to use `snap` sync to start your L2 node)
+
+## Upgrade L1 nodes
+
+Prior to the L2 hardfork, all node operators must upgrade their existing L1 ("celo-blockchain") nodes to the appropriate release version. You can find the hardfork release versions for each network listed in the Key Information section below.
+These releases each define hardfork block numbers such that nodes will stop producing or accepting blocks immediately before the hardfork.
+
+## Run a pre-migration
+
+Node operators who plan to migrate their pre-hardfork data to an L2 node are strongly encouraged to run a pre-migration ~1-2 days before the hardfork. This will migrate the majority of data and minimize downtime during the full migration. To learn more about the migration process and tooling, see [Migrating a Celo L1 Node](../operators/migrate-node.md).
+
+Note that you may also choose to start an L2 node from scratch using `snap` sync, in which case you don't need to run any data migrations. See [Running a Celo Node](../operators/run-node.md).
```

Comment:
Personal opinion: I'd put this option at the start of the section - it might encourage more people to opt for this easier and faster option